src/CMakeLists.txt

```diff
@@ -79,17 +84,21 @@ include_directories(${fmt_INCLUDE_DIRS_RET})
 include_directories(${HDF5_INCLUDE_DIRS_RET})
 include_directories(${Boost_INCLUDE_DIRS_RET})
 include_directories(${CMAKE_SOURCE_DIR}/src)
+include_directories(${primordial_chem_dirs} ${CMAKE_BINARY_DIR} "includes/extern_parameters.H" "includes/network_properties.H")
```

Comment:
Is it necessary to list the individual *.H files? I think this should work if you just list the directories containing the header files rather than the header files themselves.

Reply:
I tried but it didn't work that way

Reply:
That's strange. Where is the `includes/` directory?

Reply:
It's not a directory. I think it's some sort of CMake syntax that identifies the file being specified as being "includes" in the given directory.

Reply:
https://stackoverflow.com/questions/64669528/how-can-i-include-header-files-from-a-directory-into-cmake